website/layouts/shortcodes/githubRelRef.html

```diff
@@ -0,0 +1 @@
+/{{ replace (chomp (delimit (first 2 (split .Page.RelPermalink "/")) "" "")) "preview" .Page.Site.Params.github_branch }}/
```

Comment:
Can we leave the slashes outside, e.g. `/{{< githubRelRef > }}/`

Reply:
We can, the reason I have it like this is to match the `{{< relref >}}` that we use for the versioned links for docs pages which includes the slashes.

Reply:
Ahh gotcha